src/io/LivermoreParamsReader.hh

```diff
@@ -0,0 +1,45 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file LivermoreParamsReader.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <string>
+#include "physics/em/LivermoreParams.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Load the Livermore EPICS2014 photoelectric data.
+ */
+class LivermoreParamsReader
+{
+  public:
+    //!@{
+    //! Type aliases
+    using result_type = LivermoreParams::ElementInput;
+    //!@}
+
+  public:
+    // Construct the reader
+    explicit LivermoreParamsReader(const char* path = nullptr);
+
+    // Read the data for the given element
+    result_type operator()(ElementDefId el_id, int atomic_number);
```

Comment:
The `el_id` and `atomic_number` are kind of redundant here (see the other comment on `LivermoreParams::ElementInput`). Since the reading doesn't depend on `el_id` let's omit it.